R/ard_continuous.R

```diff
@@ -128,17 +128,28 @@ ard_continuous <- function(data,
       data = data
     )
 
+  df_var_type <- data |>
+    dplyr::select(variables) %>%
+    purrr::map_df(., class) %>%
+    tidyr::pivot_longer(.,
+                        cols = everything(),
+                        names_to = "variable",
+                        values_to = "var_type")
+
   # unnest results
   df_results <-
     df_nested |>
     dplyr::select(all_ard_groups(), "...ard_all_stats...") |>
     tidyr::unnest(cols = "...ard_all_stats...")
 
+  # is this a duplication?
+
   # add default function labels and formatters
   df_results <-
     df_nested |>
     dplyr::select(all_ard_groups(), "...ard_all_stats...") |>
-    tidyr::unnest(cols = "...ard_all_stats...")
+    tidyr::unnest(cols = "...ard_all_stats...") %>%
+    dplyr::left_join(., df_var_type, by = "variable")
```

Comment:
It's a hacky solution but I injected a `var_type` column to preserve the class of the original column, and use this to workout the logic below, your reprex should now show in the expected character format at least @ddsjoberg

Reply:
Sorry, i am a little confused. Why do we need the variable type?

Reply:
Originally I was thinking that we can just use the class/type of the returned statistic.  If the users are requesting a mean or standard deviation of a date, what are they even doing?! But something like a min or max seems ok, and those results are returned as Date class.

Reply:
So my hacky join-based solution was way to preserve the class/type somehow cuz I hadn't a clue how/where in the code it got lost in the summarization process to address this, which you certainly found a much more elegant answer to (I made my pushes like 2 minutes after this message):

"Ahhh, the ifelse() in the min and max functions were coercing the results to numeric (GAH!). I switch it to an if () else() and it seems to all work now. Thank you! "